server/ws/plugin_adapter.go

```diff
@@ -285,6 +295,27 @@ func (pa *PluginAdapter) getUserIDsForWorkspace(workspaceID string) []string {
 	return userIDs
 }
 
+func (pa *PluginAdapter) getUserIDsForAllWorkspaces() []string {
+	userMap := map[string]bool{}
+
+	for _, listener := range pa.listeners {
+		userMap[listener.userID] = true
+	}
+
+	userIDs := []string{}
+	for userID := range userMap {
+		userIDs = append(userIDs, userID)
+	}
+	return userIDs
+}
+
+func (pa *PluginAdapter) BroadcastConfigChange(pluginConfig model.ClientConfig) {
+	userIDs := pa.getUserIDsForAllWorkspaces()
+	for _, userID := range userIDs {
```

Comment:
I think you don't need this loop, I would send this to everybody, I think that is basically a empty `mmModel.WebsocketBroadcast` parameter.

Reply:
Also, this is probably better handled by the mattermost-server, just sending this to everybody.